bucket/smplayer.json

```
@@ -0,0 +1,56 @@
+{
+    "version": "22.2.0.0",
+    "description": "A video player forked from MPV. It can also play Youtube videos, search and download subtitles, and includes other features like a thumbnail generator and audio and video filters.",
+    "homepage": "https://www.smplayer.info/",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.fosshub.com/SMPlayer.html?dwl=smplayer-portable-22.2.0.0-x64.7z",
+            "hash": "c054f61eddc3ac97b0ce03c3001e6e080a9e6543de999a071df17e2a7fe3618d"
+        },
+        "32bit": {
+            "url": "https://www.fosshub.com/SMPlayer.html?dwl=smplayer-portable-22.2.0.0-win32-qt5.6.7z",
+            "hash": "be10cdc34a9b6282e1e91e9b789825a11939576750f1d5be839e2a4892532402"
+        }
+    },
+    "pre_install": [
+        "Rename-Item \"$dir\\SMPlayer.html\" 'dl.7z'",
+        "Expand-7zipArchive \"$dir\\dl.7z\" \"$dir\" -ExtractDir 'smplayer-portable' -Removal | Out-Null",
+        "'hdpi.ini', 'playlist.ini', 'favorites.m3u8', 'radio.m3u8', 'tv.m3u8' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" }",
```

```suggestion
        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
```
To make creating new ini files silent

---

Oh. I missed that. Thank you for reminding me.